crates/re_data_source/src/data_loader/loader_archetype.rs

```diff
@@ -0,0 +1,157 @@
+use re_log_types::{DataRow, EntityPath, RowId, TimePoint};
+
+use crate::{DataLoader, DataLoaderError, LoadedData};
+
+// ---
+
+/// Loads data from any supported file or in-memory contents as native [`re_types::Archetype`]s.
+pub struct ArchetypeLoader;
```

Comment:
I don't really follow the "archetype" part of this. As opposed to what? Data that cannot be loaded as archetypes?

Is this specifically for things that have a 1-to-1 mapping with archetypes, i.e. one file = one archetype instance?

Reply:
> Is this specifically for things that have a 1-to-1 mapping with archetypes, i.e. one file = one archetype instance?

Yeah that was my thought pretty much, I can document it this way.